src/elements/content-sidebar/SidebarPanels.js

```diff
@@ -266,7 +266,7 @@ class SidebarPanels extends React.Component<Props, State> {
                                     hasSidebarInitialized={isInitialized}
                                     ref={this.boxAISidebar}
                                     startMarkName={MARK_NAME_JS_LOADING_BOXAI}
-                                    userInfo={{name: currentUser?.name, avatarUrl: currentUser?.avatar_url}}
+                                    userInfo={{name: currentUser?.name, avatarURL: currentUser?.avatar_url}}
```

Comment:
there seems to be missing formatting:
```suggestion
                                    userInfo={{ name: currentUser?.name, avatarURL: currentUser?.avatar_url }}
```